database/connection.go

```diff
@@ -0,0 +1,42 @@
+package database
+
+import (
+	"log"
+	"os"
+	"sync"
+
+	"gopkg.in/jackc/pgx.v2"
```

Comment:
Avoid pgx, just use libpq for now. Keep the code database agnostic as long as possible - we may need to support mysql as well.